rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -188,6 +241,31 @@ impl Layout {
         LIVESPLIT_STATE.lock().unwrap().layout.general_settings().clone()
     }
 }
+impl LiveSplitTime {
+    pub fn get_digits_format_and_accuracy() -> (LiveSplitDigitsFormat, LiveSplitAccuracy) {
+        let livesplit_state = LIVESPLIT_STATE.lock().unwrap();
+        (livesplit_state.digits_format, livesplit_state.accuracy)
+    }
+    pub fn set_digits_format_and_accuracy(digits_format: LiveSplitDigitsFormat, accuracy: LiveSplitAccuracy) {
+        let mut livesplit_state = LIVESPLIT_STATE.lock().unwrap();
+        livesplit_state.digits_format = digits_format;
+        livesplit_state.accuracy = accuracy;
+    }
+    pub fn format(ts: TimeSpan) -> String {
+        let digits_format_accuracy = Self::get_digits_format_and_accuracy();
+        let time = LiveSplitTime {
+            time: Time::with_digits_format(digits_format_accuracy.0),
+            fraction: Fraction::with_accuracy(digits_format_accuracy.1),
+        };
+        time.format(ts).to_string()
+    }
```

Comment:
This function isn't needed with the previous improvement. You can use `TimeFormatter::format` directly on `LiveSplit.time`.